lib/pages/new_private_chat/widget/loading_contact_widget.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:flutter/material.dart';
+
+class LoadingContactWidget extends StatelessWidget {
+  const LoadingContactWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Column(
+        children: [
+          const CircularProgressIndicator(),
```

Comment:
please add more gap from the appbar by adding above:
```
const SizedBox(height: 16.0,),
```

Reply:
Done!